docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -1,52 +1,73 @@
 # How to Setup a small Dev Cluster of multi-nodes
```

Comment:
I suggest removing "small" from the title. So we dont' have deal with questions like "how small is regarded as small" or "what is the limit".

Reply:
good point. will remove it.